.github/workflows/daily-graalvm-image.yml

```diff
@@ -41,7 +41,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: graalvm/setup-graalvm@v1
         with:
-          version: '22.2.0'
+          version: 'dev'
```

Comment:
- Switched from https://github.com/graalvm/graalvm-ce-builds to https://github.com/graalvm/graalvm-ce-dev-builds for faster resolution of earlier issues.